generators/app/__snapshots__/generator.spec.mts.snap

```diff
@@ -145,6 +144,7 @@ Object {
   "devDatabaseTypeOracle": false,
   "devDatabaseTypePostgres": false,
   "devServerPort": 4200,
+  "dockerServicesDir": "src/main/docker/",
```

Comment:
I always wondered, why we put the compose files under src/main. They are not src files imho.

Reply:
Seems that by default all maven plugins that create images use / or src/main/docker.

This is the case for:
- https://github.com/spotify/docker-maven-plugin
- https://github.com/spotify/docker-client
- https://github.com/fabric8io/docker-maven-plugin
- https://www.eclipse.org/jkube/docs/openshift-gradle-plugin#dockerfile-scenario

jib-maven-plugin uses src/main/jib

Even quarkus use that path too:
- https://quarkus.io/guides/building-native-image#manually-using-the-micro-base-image

Reply:
We have different purposes files there.
The docker-compose files and the files related to Dockerfile used to build the docker image.
IMO docker compose files would fit better at `/docker`
While `src/main/docker/jib/*` files are in right place.

Would 2 directory for docker related files confuse users?